src/styles/index.scss

```diff
@@ -1,3 +1,139 @@
-body {
-  margin: 0;
-}
+  @import '/src/styles/stars';
+
+  * {
+    box-sizing: border-box;
+  }
+
+  html {
+    font-family: Roboto, sans-serif;
+  }
+
+  body,
+  h2,
+  p,
+  div {
+    margin: 0;
+  }
+
+  .card {
+    padding-inline: 16px;
+    padding: 32px 0 16px;
+    width: 200px;
+    height: 406px;
+    flex-shrink: 0;
+    border-radius: 5px;
+    border: 1px solid #f3f3f3;
+    background: #fff;
+
+
+    &__image {
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      margin: 0 auto 40px;
+      width: 160px;
+      height: 134px;
+      background-image: url(../images/imac.jpeg);
+      background-size: cover;
+      background-position: center;
+    }
+
+    &__header,
+    &__product-code,
+    &__reviews-container,
+    &__price-container,
+    &__button {
+      margin-left: 16px;
+    }
+
+    &__header {
+      margin-bottom: 4px;
+      width: 166px;
+      color:  #060b35;
+      font-style: normal;
+      font-size: 12px;
+      font-weight: 500;
+      line-height: 18px;
+    }
+
+    &__product-code {
+      margin-bottom: 16px;
+      color:  #616070;
+      font-style: normal;
+      font-size: 10px;
+      font-weight: 400;
```

Comment:
Its default values of `font-style` and `font-weight` - redundant
```suggestion
      font-size: 10px;
```

Reply:
Still not fixed from the previous review
<img width="813" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/64750958/41a293a4-b47c-42c5-aadc-eff571221739">